README.md

```diff
@@ -2,7 +2,7 @@
     Bert Verstraete    
     CMS   
     2018-2019  
-    Bachlor Applied Computer Science  
+    Bachelor Applied Computer Science  
     Howest College University
 ## Deployment
-    // TODO
\ No newline at end of file
+    // TODO
```

Comment:
No new line at end of file, ...

Reply:
PP: I prefer to not have a newline, thanks for the suggestion though, Yarne. 🙃